src/main/resources/swagger/rawls.yaml

```diff
@@ -227,6 +227,167 @@ paths:
             - openid
             - email
             - profile
+  '/admin/groups':
+      post:
+        responses:
+          '201':
+            description: Created
+          '403':
+            description: Unauthorized
+            schema:
+              $ref: '#/definitions/ErrorReport'
+          '500':
+            description: Rawls Internal Error
+            schema:
+              $ref: '#/definitions/ErrorReport'
+        description: Create a group
+        tags:
+          - admin
+        summary: create a group
+        operationId: adminCreateGroup
+        parameters:
+          - in: body
+            description: RawlsGroupRef
```

Comment:
I thought users were going to be referring to other users and groups by email, not by subject id?